release_notes.md

```diff
@@ -0,0 +1,12 @@
+# Loop Client Release Notes
+This file tracks release notes for the loop client. 
+
+Developers: 
```

Comment:
Consider making **Developers** and **Release Manager** bold to stand out better.